src/components/BenefitPackageIndividualPanel.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import {
+  Grid, Typography, IconButton, Tooltip,
+} from '@material-ui/core';
+import {
+  FormattedMessage,
+  PublishedComponent,
+  TextInput,
+  NumberInput,
+  FormPanel,
+  formatMessage,
+} from '@openimis/fe-core';
+import { Person } from '@material-ui/icons';
+import { injectIntl } from 'react-intl';
+import { withTheme, withStyles } from '@material-ui/core/styles';
+import { EMPTY_STRING, RIGHT_INDIVIDUAL_UPDATE } from '../constants';
+
+const styles = (theme) => ({
+  tableTitle: theme.table.title,
+  item: theme.paper.item,
+  paper: theme.paper.paper,
+  fullHeight: {
+    height: '100%',
+  },
+});
+
+function renderHeadPanelSubtitle(rights, intl, history, modulesManager, classes, individualUuid) {
+  const openIndividual = () => history.push(`/${modulesManager.getRef('individual.route.individual')}`
+    + `/${individualUuid}`);
+
+  return (
+    <Grid item>
+      <Grid container align="center" justify="center" direction="column" className={classes.fullHeight}>
+        <Grid item>
+          <Typography>
+            <FormattedMessage
+              module="socialProtection"
+              id="socialProtection.benefitPackage.IndividualDetailPanel.title"
+            />
+            { !!individualUuid && (
+            <Tooltip title={formatMessage(intl, 'socialProtection', 'benefitPackage.IndividualDetailPanel.tooltip')}>
+              <IconButton onClick={openIndividual} disabled={!rights.includes(RIGHT_INDIVIDUAL_UPDATE)}>
+                <Person />
+              </IconButton>
+            </Tooltip>
+            )}
+          </Typography>
+        </Grid>
+      </Grid>
+    </Grid>
+  );
+}
+
+class BenefitPackageIndividualPanel extends FormPanel {
+  // eslint-disable-next-line class-methods-use-this
+  createAdditionalField(jsonExt) {
+    if (!jsonExt) return [];
+
+    const additionalFields = JSON.parse(jsonExt);
+
+    const arrayWithFields = Object.entries(additionalFields).map(([property, value]) => ({
+      field: { [property]: value },
+      fieldType: typeof value,
+    }));
+
+    return arrayWithFields;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  renderProperType({ fieldType, field }) {
```

Comment:
I wonder if should we use here case switch construction or move to simply construction with using if/else etc. 
```
renderProperType({ fieldType, field }) {
  const NUMBER = 'number';
  const { 0: label, 1: value } = Object.entries(field)[0];

  if (fieldType === NUMBER) {
    return (
      <NumberInput
        module="socialProtection"
        readOnly
        min={0}
        displayZero
        label={label}
        value={value}
      />
    );
  }

  return (
    <TextInput
      module="socialProtection"
      readOnly
      label={label}
      value={value}
    />
  );
}
```